### ISSUE TITLE
Question about training settings and data preprocessing

### ISSUE THREAD

**Issue by flying0712** [LabShuHangGU/DCAE#5]
Hi, thank you for sharing this excellent work. I tried to reproduce the results following the paper’s experimental settings (*Training Details* section). However, my reproduced results differ significantly in terms of bitrate.

* **Paper (pretrained model, MSE loss, λ=0.05 on CLIC2020):**

  * average\_PSNR: 38.58 dB
  * average\_MS-SSIM: 20.5027
  * average\_Bit-rate: 0.594 bpp

* **My re-training (same setup):**

  * average\_PSNR: 38.56 dB
  * average\_MS-SSIM: 20.4508
  * average\_Bit-rate: 0.731 bpp

As you can see, the reconstruction quality is very close, but the bitrate is more than 20% higher.

My training command was:

```bash
CUDA_VISIBLE_DEVICES='0,1,2,3,4,5' \
python -m torch.distributed.launch --nproc_per_node=6 --master_port=29506 train.py \
    -d /data/dataset/openimages-300k/ \
    -lr 1e-4 \
    --cuda \
    --lambda 0.05 \
    --epochs 50\
    --lr_epoch 46 \
    --batch-size 8 \
    --save_path ./checkpoints
```

The training dataset was 300k images randomly selected from OpenImages, each with width and height ≥256.

I would like to ask:

1. Is there any preprocessing requirement beyond what is described in the paper (e.g., specific filtering or normalization of images)?
2. Could you share the exact **list of training image files** used?
3. Are there any training tricks or implementation details not mentioned in the paper that are critical for reproducing the reported bitrate?
4. For the `--lambda` parameter, I directly followed the paper’s values `{0.0018, 0.0035, 0.0067, 0.0130, 0.0250, 0.0500}` (e.g., my train with `--lambda 0.05` corresponds to the last value). Could you confirm that this is the correct way to set it?

Thanks a lot for your time and clarification!
